configuration/network-bridge-support.md

```diff
@@ -42,12 +42,12 @@ First, retrieve the attachment of this Wifi article in dom0. Then apply the thre
 
 Finally restart the qubes manager GUI.
 
-A new option is now available in the AppVM Settings to enable set the NetVM in bridge mode. For a bridged AppVM, you should the select a netvm instead of a firewall vm, enabled the Bridge option and restart your AppVM.
+A new option is now available in the AppVM Settings to enable set the NetVM in bridge mode. For a bridged AppVM, you should the select a netvm instead of a firewall vm, enable the Bridge option and restart your AppVM.
```

Comment:
"An option is available in the AppVM settings to enable setting the NetVM in bridge mode. For a bridged AppVM, you should the select a NetVM instead of a FirewallVM / ProxyVM, enable the Bridge option, and restart your AppVM."?